Add note about not supporting multiple clouds in the dashboards (#272)
infrawatch/documentation: doc-Service-Telemetry-Framework/modules/proc_importing-dashboards.adoc

[role="_abstract"]
The Grafana Operator can import and manage dashboards by creating `GrafanaDashboard` objects. You can view example dashboards at https://github.com/infrawatch/dashboards.

> WARNING: The dashboards provided via https://github.com/infrawatch/dashboards are currently only compatible with a single cloud. Using the dashboards when multiple clouds have been configuration or changing the address on the overcloud that metrics are sent can result in errors when running the queries against Prometheus, such as `found duplicate series` errors.

Reviewer: ```suggestion
WARNING: The dashboards are currently only compatible with a single cloud. If you use the dashboards when multiple clouds have been configured or if you change the address on the overcloud to which that metrics are sent, this can result in errors when you run the queries against Prometheus, such as `found duplicate series` errors.
```
